modules/ibi-create-config-iso.adoc

```diff
@@ -0,0 +1,316 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-config-iso_{context}"]
+= Deploying a managed {sno} cluster using the IBI Operator
+
+Create the site-specific configuration resources in the hub cluster to initiate the image-based deployment of a preinstalled host. 
+
+When you create these configuration resources in the hub cluster, the Image Based Install (IBI) Operator generates a configuration ISO and attaches it to the target host to begin the site-specific configuration  process. When the configuration process completes, the {sno} cluster is ready.
+
+[NOTE]
+====
+For more information about the configuration resources that you must configure in the hub cluster, see "Cluster configuration resources for deploying a preinstalled host".
+====
+
+.Prerequisites
+
+* You preinstalled a host with {sno} using an image-based installation.
+* You logged in as a user with `cluster-admin` privileges.
+* You deployed a {rh-rhacm-first} hub cluster or you deployed the multicluster engine for Kubernetes operator (MCE).
+* You installed the IBI Operator on the hub cluster.
+* You created a pull secret to authenticate pull requests. See the "Additional resources" section for more information about using image pull secrets.
+
+.Procedure
+
+. Create the `ibi-ns` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create namespace ibi-ns
+----
+
+. Create the `Secret` resource for your image registry:
+
+.. Create a YAML file that defines the `Secret` resource for your image registry:
++
+.Example `secret-image-registry.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ibi-image-pull-secret
+  namespace: ibi-ns
+stringData:
+  .dockerconfigjson: <base64-docker-auth-code> <1>
+type: kubernetes.io/dockerconfigjson
+----
+<1> You must provide base64-encoded credential details. See the "Additional resources" section for more information about using image pull secrets.
+
+.. Create the `Secret` resource for your image registry by running the following command:
++
+[source,terminal]
+----
+$ oc create -f secret-image-registry.yaml
+----
+
+. Optional. Configure static networking for the host:
+
+.. Create a `Secret` resource containing the static network configuration in `nmstate` format:
++
+.Example `host-network-config-secret.yaml` file
+[source,yaml]
+----
+apiVersion: v1 
+kind: Secret
+metadata:
+ name: host-network-config-secret <1>
+ namespace: openshift-machine-api
```

Comment:
It's more common to see this secret created in the same namespace as the other resources (ibi-ns in this example)